legion/engine/physics/physics_statics.cpp

```diff
@@ -105,22 +128,21 @@ namespace legion::physics
                         if (attemptBuildMinkowskiFace(edgeA, edgeB, transformA, transformB))
                         {
                             //get world edge direction
-                            math::vec3 edgeADirection = transformA * math::vec4(edgeA->getLocalEdgeDirection(), 0);
-
-                            math::vec3 edgeBDirection = transformB * math::vec4(edgeB->getLocalEdgeDirection(), 0);
+                            math::vec3 edgeADirection = transformA * math::vec4(edgeA->getRobustEdgeDirection(), 0);
+                            math::vec3 edgeBDirection = transformB * math::vec4(edgeB->getRobustEdgeDirection(), 0);
 
-                            edgeADirection = math::normalize(edgeADirection);
-                            edgeBDirection = math::normalize(edgeBDirection);
+                            edgeADirection = math::normalize( edgeADirection );
+                            edgeBDirection = math::normalize( edgeBDirection );
 
-                            //get the seperating axis
-                            math::vec3 seperatingAxis = math::cross(edgeADirection, edgeBDirection);
+                            static float angleThres = math::cos(math::deg2rad(3.0f));
```

Comment:
i do recall that I wanted this to be a constexpr value in physics::constants, but this wasn't possible because there is no constexpr version of math::cos. I could just declare the literal value of math::cos(math::deg2rad(3.0f)) in physics::constants, is that sufficient?

Reply:
yeah if you can't make it `constexpr`, it's fine to make it `static const`. it won't be generated on compile time, but it will be generated at startup.